rclpy/rclpy/node.py

```diff
@@ -30,13 +33,35 @@
 
 class Node:
 
-    def __init__(self, handle):
+    def __init__(self, node_name, *, namespace=None):
         self.clients = []
-        self._handle = handle
+        self._handle = None
         self.publishers = []
         self.services = []
         self.subscriptions = []
         self.timers = []
+        self._default_callback_group = MutuallyExclusiveCallbackGroup()
+
+        namespace = namespace or ''
+        failed = False
+        if not ok():
+            raise NotInitializedException('cannot create node')
+        try:
+            node_handle = _rclpy.rclpy_create_node(node_name, namespace)
+            self._handle = node_handle
+        except ValueError:
+            failed = True
+        if failed:
+            # these will raise more specific errors if the name or namespace is bad
+            validate_node_name(node_name)
+            # emulate what rcl_node_init() does to accept '' and relative namespaces
+            if not namespace:
+                namespace = '/'
+            if not namespace.startswith('/'):
+                namespace = '/' + namespace
+            validate_namespace(namespace)
+            # Should not get to this point
+            raise RuntimeError("rclpy_create_node failed for unknown reason")
```

Comment:
This should be single quotes. I would assume if you re-run CI builds they should flag that (as of ament/ament_lint#84).

Reply:
Fixed in f2c23341c419c0cc652252e11d042770a652aae4